bin/cli.js

```diff
@@ -65,18 +63,13 @@ const cliFlags = yargs
     return true;
   }).argv;
 
-//Get all explicitly terminal set options, does not include url because url has no default option
```

Comment:
@paulirish, @denar90's train of thought is similar to my own.
My original idea was to be able to unify all options.  Lets say as a user I don't want to type out all explicit flags I need. To solve this I set up a config.file with my intended options.

If for some reason I want to test something out and don't want or can't (current work condition) I want to be able to explicitly overide that specific option in cli.

This approach was the way I found to make this precedence order function without breaking too much:
 - Get everything expliticly set in the command line.
 - Get everything from the config file
 - Whatever has not be yet set is the default value (which yargs defines).

I could have requested a change or submitted a pull request for lighthouse directly but my use case for ```lighthouse``` is merely for performance at the moment and in a visual manner (which ```pwmetrics``` is perfect for). Though to be honest the amount of lighthouse cli options daunts me 😛